graph/src/data/query/error.rs

```diff
@@ -151,6 +152,9 @@ impl fmt::Display for QueryExecutionError {
             StoreError(e) => {
                 write!(f, "Store error: {}", e)
             }
+            MissingNameInTypeIntrospection => {
```

Comment:
This is `MissingArgumentError`, we don't have to invent a more specific one here IMHO.